src/openrct2/network/NetworkBase.cpp

```diff
@@ -815,6 +817,22 @@ void NetworkBase::KickPlayer(int32_t playerId)
     }
 }
 
+int32_t NetworkBase::GetTotalPlayerCount() const
+{
+    return player_list.size();
+}
+
+int32_t NetworkBase::CountVisiblePlayers() const {
+    int32_t result = 0;
+    
+    for (const auto& player : player_list) {
+        if (!(player->Flags & NETWORK_PLAYER_FLAG_ISHIDDEN))
+            result++;
+    }
+    
+    return result;
+}
```

Comment:
I had considered caching the amount of visible players in a variable in NetworkBase, but that would be vulnerable to getting out of sync with player_list, so this was the safer option.
Also the variables in NetworkBase were nicely aligned which made it less appealing to stuff another variable inbetween them and ruin the alignment.
Normally there's less than ten players on a server and the maximum supported player count also isn't that high as far as I'm aware, so the cost of this function should be very very low.